spec/common/participants.html

```diff
@@ -1 +1 @@
-Members of the RDF-star Working Group Group included Achille Zappa, Adrian Gschwend, Andy Seaborne, Antoine Zimmermann, Dan Brickley, David Chaves-Fraga, Dominik Tomaszuk, Dörthe Arndt, Enrico Franconi, Fabien Gandon, Gregg Kellogg, Gregory Williams, Jesse Wright, Jose Emilio Labra Gayo, Julián Arenas-Guerrero, Olaf Hartig, Ora Lassila, Pasquale Lisena, Peter Patel-Schneider, Pierre-Antoine Champin, Raphaël Troncy, Ruben Taelman, Rémi Ceres, Souripriya Das, Stuart Sutton, Ted Thibodeau, Thomas Pellissier Tanon, Timothée Haudebourg, and Vladimir Alexiev.
+Members of the RDF-star Working Group Group included Achille Zappa, Adrian Gschwend, Alan Snyder, Amin Anjomshoaa, Andy Seaborne, Antoine Zimmermann, Dan Brickley, Dave Raggett, Dominik Tomaszuk, Dörthe Arndt, Enrico Franconi, Erich Bremer, Fabien Gandon, Felix Sasaki, Gregg Kellogg, Gregory Williams, Jean-Yves Rossi, Jose Emilio Labra Gayo, Julián Arenas-Guerrero, Kurt Cagle, Niklas Lindström, Olaf Hartig, Ora Lassila, Pasquale Lisena, Peter Patel-Schneider, Pierre-Antoine Champin, Raphaël Troncy, Richard Lea, Ruben Taelman, Rémi Ceres, Souripriya Das, Ted Thibodeau Jr, Thomas Lörtsch, Thomas Pellissier Tanon, Timothée Haudebourg, and Vladimir Alexiev.
```

Comment:
This puts one name per line, and adds whitespace that will be folded when browsers display the HTML, for easier visual ordering. Lists of names are *usually* sorted by family name, which I've applied below. Whatever the ordering method, it should be consistent, and generally treat accented letters as if unaccented (e.g., `Rémi` should be ordered as if `Remi`).
```suggestion
Members of the RDF-star Working Group Group included 
      Vladimir Alexiev,
          Amin Anjomshoaa,
        Julián Arenas-Guerrero,
        Dörthe Arndt,
         Erich Bremer,
           Dan Brickley,
          Kurt Cagle,
          Rémi Ceres,
Pierre-Antoine Champin,
    Souripriya Das,
        Enrico Franconi,
        Fabien Gandon,
        Adrian Gschwend,
          Olaf Hartig,
      Timothée Haudebourg,
         Gregg Kellogg,
   Jose Emilio Labra Gayo,
           Ora Lassila,
       Richard Lea,
        Niklas Lindström,
      Pasquale Lisena,
        Thomas Lörtsch,
         Peter Patel-Schneider,
        Thomas Pellissier Tanon,
          Dave Raggett,
     Jean-Yves Rossi,
         Felix Sasaki,
          Andy Seaborne,
          Alan Snyder,
         Ruben Taelman,
           Ted Thibodeau Jr,
       Dominik Tomaszuk,
       Raphaël Troncy,
       Gregory Williams,
       Achille Zappa,
and
       Antoine Zimmermann.
```

Reply:
The list is generated using the W3C API via https://github.com/w3c/rdf-common/blob/main/participants.js. They are responsible for the ordering, so if it is not properly ordered, the change should happen in that script. I have no interest in manually managing the participant list. But, it's about time to re-generate that list as we have some new participants that aren't captured in the current export.

My process has been to run the script in the rdf-common repository and then create PRs for each of the main repositories with copy of the export.

Reply:
Actually, looking at the script, the results are sorted using the title field from the users list. We could potentially be smarter about how to do the sort, but at the expense of some name format assumptions. The API endpoint that returns the users is `https://api.w3.org/groups/wg/rdf-star/users?items=50` which current yields the following results:

```json
{
    "page": 1,
    "limit": 50,
    "pages": 1,
    "total": 43,
    "_links": {
        "up": {
            "href": "https://api.w3.org/groups/wg/rdf-star"
        },
        "users": [
            {
                "href": "https://api.w3.org/users/9t56qgfqo34s88cwg8gc0wsskwcw84g",
                "title": "Vladimir Alexiev",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/pectasibqkg4wkgs4c0o84wgg4wwk48",
                "title": "Amin Anjomshoaa",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/k16qex1vxs0wwog4scwg0kow8kg0kkw",
                "title": "Julián Arenas-Guerrero",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/jwtjh0djnaoos00c00cw4kg4kgwsw0k",
                "title": "Dörthe Arndt",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/gro4rj47iy8sckwgkcs8k40c4o0wgk4",
                "title": "Bilal Ben Mahria",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/1xl67nr2zk9ww8cwo0g0wg0k0ggs0wg",
                "title": "Erich Bremer",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/9a26fdb3yqskoks88gkw4gwocscgk4o",
                "title": "Kurt Cagle",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/cbrhhie7588w080kcss80oscc4ggks",
                "title": "Rémi Ceres",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/t891ludoisggsccsw44o8goccc0s0ks",
                "title": "Pierre-Antoine Champin",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/k557snqe1j4wsw0g4gksw0wwg8oscok",
                "title": "Souripriya Das",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/szv8llvlksg0wwkowok0so0kcks8sws",
                "title": "Daniil Dobriy",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/s5qsla39msggw4occg484g0o04sk4kw",
                "title": "Enrico Franconi",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/c5gmgoyed9cgw88w88ko4swg4ss0000",
                "title": "Jeffrey Phillips Freeman",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/1ebxwvxr95y8c00kc88osw00048k84k",
                "title": "Fabien Gandon",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/3l06p756yq040g0goko0wws4gwkoco4",
                "title": "Benjamin Goering",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/4riljs6mjk004sgw0cgw8w08k4sw84s",
                "title": "Adrian Gschwend",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/jjjsv5p2m3w4w0oc84wss8g848wo0gg",
                "title": "Olaf Hartig",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/35j4kp2lrw6cs48ks8w0k4s80kgo88w",
                "title": "Timothée Haudebourg",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/rqjupo9k2uos0480ks0c0w0oc84csos",
                "title": "Ian Horrocks",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/8nomt3nm1n8cw4s44koggo4wcoo8w4s",
                "title": "Gregg Kellogg",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/acln4jq5pxs8sc44wo0c0o4sg4kwcss",
                "title": "Mark Kim",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/qf7jsf8u7cgokg0csk00co40gc0ckgw",
                "title": "Jose Emilio Labra Gayo",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/tkgxmc97wpw448k844kwwogsk0cwckw",
                "title": "Ora Lassila",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/fselkewnbpc0skko0os8soco8ks0osk",
                "title": "Richard Lea",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/j1abbgdrjtc8o0oksokos4840c0okgo",
                "title": "Niklas Lindström",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/jhzm0uqjteogos0sk0cgs44g04ccsss",
                "title": "Pasquale Lisena",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/k4eyq9e00dssws4ggccoccgcwooc40c",
                "title": "Thomas Lörtsch",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/sbvn94wkykg00oswcgg8c4sggwwcc4k",
                "title": "Matthew Nguyen",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/51kr2s2vw9c8wgssc8ookg88488s8ks",
                "title": "Peter Patel-Schneider",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/5l351hvf6kw848c0ooo884c8kcgoggk",
                "title": "Thomas Pellissier Tanon",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/ksgplwkb6pco08c0cw0okg408oggkss",
                "title": "Dave Raggett",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/qg3tx00tj2840kkgg0cc084wgwcwsk",
                "title": "Jean-Yves ROSSI",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/lr343j5229cs4k8kcsoc4ww0c8cks80",
                "title": "Felix Sasaki",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/ez5whckagcg0cog0wg8occsgggoog8c",
                "title": "Andy Seaborne",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/ocafw9i2a348g8okg0c0oog8kck4s4w",
                "title": "Ruben Taelman",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/ox8xc2908xcowo8wwkog0cgokcw0o8o",
                "title": "Ted Thibodeau Jr",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/p2qc2g608rkkc8w8okowokk40g4s8sg",
                "title": "Dominik Tomaszuk",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/ladv9x6f1sgcwc88o80k08w8gco84cg",
                "title": "Raphaël Troncy",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/7tocy07hy7ks0s8cosck08c08k0gsk0",
                "title": "William Van Woensel",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/4l7bci6vxq4gcwc448kwwogw48ocw0k",
                "title": "Gregory Williams",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/b39dpa6cki88csg4ko0scgc0w8oc884",
                "title": "Jesse Wright",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/behozvazzio8808g0owksksk8s8884k",
                "title": "Achille Zappa",
                "former": false
            },
            {
                "href": "https://api.w3.org/users/m1r1atazz8gg0cwsggc0wk8o4wc0oks",
                "title": "Antoine Zimmermann",
                "former": false
            }
        ],
        "self": {
            "href": "https://api.w3.org/groups/wg/rdf-star/users?page=1&items=50"
        },
        "first": {
            "href": "https://api.w3.org/groups/wg/rdf-star/users?page=1&items=50"
        },
        "last": {
            "href": "https://api.w3.org/groups/wg/rdf-star/users?page=1&items=50"
        }
    }
}
```

The script extracts the 'title' field from each user record and sorts them. It looks like are already sorted by family name, so we can probably just remove the sort operation in the script and use the natural result. That would give the following result:

  Members of the RDF-star Working Group Group included Vladimir Alexiev, Amin Anjomshoaa, Julián Arenas-Guerrero, Dörthe Arndt, Bilal Ben Mahria, Erich Bremer, Kurt Cagle, Rémi Ceres, Pierre-Antoine Champin, Souripriya Das, Daniil Dobriy, Enrico Franconi, Jeffrey Phillips Freeman, Fabien Gandon, Benjamin Goering, Adrian Gschwend, Olaf Hartig, Timothée Haudebourg, Ian Horrocks, Gregg Kellogg, Mark Kim, Jose Emilio Labra Gayo, Ora Lassila, Richard Lea, Niklas Lindström, Pasquale Lisena, Thomas Lörtsch, Matthew Nguyen, Peter Patel-Schneider, Thomas Pellissier Tanon, Dave Raggett, Jean-Yves ROSSI, Felix Sasaki, Andy Seaborne, Ruben Taelman, Ted Thibodeau Jr, Dominik Tomaszuk, Raphaël Troncy, William Van Woensel, Gregory Williams, Jesse Wright, Achille Zappa, and Antoine Zimmerman

Reply:
I'm resolving this conversation in favour of https://github.com/w3c/rdf-common/pull/12. As it also states, when that lands this and other dependent repos have to be updated to pass the checks.